### PR TITLE
fix/underline-current-page

### LINKED ISSUE #191: Highlight which page the user is on always
### Feature Description

_A clear and concise description of the feature you're requesting._

Always underline which container page the user is on (so events, people, legislation, etc.)

### Use Case

_Please provide a use case to help us understand your request in context._

"The user should always know where they are on the site" - Pia

### Solution

_Please describe your ideal solution._

### Alternatives

_Please describe any alternatives you've considered, even if you've dismissed them._

### Screenshots/References

_In the case of a visually oriented feature or component request, providing drawings, screenshots, and/or links to examples of what you're seeking will be very helpful._
image

Current ^^^

Proposed:
image

(the underline stays!)

### PR DESCRIPTION
**Pull request by NeelApte**
### Link to Relevant Issue

This pull request resolves #191

### Description of Changes

Underline works for current page. 

_Include a description of the proposed changes._

### Link to Forked Storybook Site

https://neelapte.github.io/cdp-frontend/

_If component changes (especially visual changes) are contained in this PR, we ask that you provide a link to a publicly viewable version of the Storybook docs site, so PR reviewers can see your changes without having to install and view your code locally._

_Please see `CONTRIBUTING.md` for directions on how this can be done._

### CODE REVIEW

**Comment by BrianL3** [comment]
This is Neel's first PR to the project.  Go Neel, thanks for bearing with me today.

File: src/components/Layout/Header/Header.tsx [hunk 1]
```diff
@@ -40,9 +40,22 @@ export interface HeaderProps {
...
+    return currUnderline;
+  }
+
+  return {};
+}
+
 const Header: FC<HeaderProps> = ({ municipalityName }: HeaderProps) => {
   const [navigationIsVisible, setNavigationIsVisible] = useState<boolean>(false);
 
+  const location = useLocation().pathname;
```
Comment (tohuynh) [inline src/components/Layout/Header/Header.tsx:57]:
```suggestion
  const pathName = useLocation().pathName;
```

**Review by tohuynh — APPROVED** [review]
Thanks! Look good to me.

**Review by JacksonMaxfield — APPROVED** [review]
Looks good to me as well! Thank you so much!

**Pull request merged**